input/data/2014/2014ApJ...781L..11A/tev-000025-sed-1.ecsv

```diff
@@ -0,0 +1,32 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_ref, datatype: float32, unit: TeV}
+# - {name: e_min, datatype: float32, unit: TeV}
+# - {name: e_max, datatype: float32, unit: TeV}
+# - {name: e2dnde, datatype: float32, unit: cm-2 s-1 TeV}
```

Comment:
here in `input/data/2014/2014ApJ...781L..11A/tev-000025-sed-1.ecsv` could you please write the unit as `TeV cm-2 s-1` instead of `cm-2 s-1 TeV`? It's the same, but for `e2dnde` we always use this form and I find it easier to read, because more different from the `cm-2 s-1 TeV-1` we use everywhere for `dnde`.